teapots/choreographer-30fps/src/main/cpp/CMakeLists.txt

```diff
@@ -17,36 +17,53 @@
 cmake_minimum_required(VERSION 3.4.1)
```

Comment:
version probably earlier than 3.6.2:  when I built with 3.6.2, it complaint that I only have 3.6.0-rc1.  The first time inside SDK maybe earlier than that, this C side code does not require 3.6.2, it certainly could build ( I could not change to 3.6.2, even I could not build with that version )

Reply:
I'm still on etcha-sketch so can't test this, but this might be not inclusive minimum.  I.e. set it to one less as it expands to error if version is equal to or less than.
I think 3.6.1 is correct here.

Reply:
when CMake was supported, the version is 3.6?

Reply:
https://b.corp.google.com/issues/37102629
Looks like the first time we support CMake is 3.4.1.   I would like to stay as is so different Android Studio version could still compile the code.  Thx